backend/src/core/models/games.aggregate.js

```diff
@@ -0,0 +1,50 @@
+import { Game } from "./game.js";
+
+export class GamesAggregate {
+  games;
```

Comment:
private

Add getter method to return games and name it `get values OR get content` which should return a copy of the private games array. Then the client won't be able to change our aggregate ;). That is the idea behind the aggregate class and encapsulation in general.